src/registry/currency/edit.tsx

```diff
@@ -159,6 +159,20 @@ const DefaultValue: React.FC = () => {
   );
 };
 
+const ComparisonValueInput: React.FC = () => {
+  return (
+    <NumberField
+      name="conditional.eq"
```

Comment:
with my above suggestions, we don't need to duplicate the form field name and literalsf or label/tooltip anymore :)